src/libecalc/core/consumers/compressor/component.py

```diff
@@ -107,16 +104,12 @@ def evaluate(
             else:
                 aggregated_result.extend(model_result)
 
-        energy_usage = TimeSeriesRate(
+        energy_usage = TimeSeriesStreamDayRate(
             values=aggregated_result.energy_usage,
             timesteps=evaluated_timesteps,
             unit=aggregated_result.energy_usage_unit,
-            regularity=evaluated_regularity,
         )
 
-        if energy_usage.unit == Unit.STANDARD_CUBIC_METER_PER_DAY:
```

Comment:
to be handled in application layer? Needed? Nice to have what it was...but perhaps we shouldnt have different defaults?

Reply:
Since users can change it in web it would be nice if we could have stream day for all. It would affect the csv though